### ISSUE TITLE
"FAILURE IN SERVER /s_new duplicate node ID"

### ISSUE THREAD

**Issue by prko** [supercollider/supercollider#3303]
Bug description:
Each time whenever launching SC with the "startup.scd" in the first step in the "Steps to reproduce", I see the following error exactly four times.
"FAILURE IN SERVER /s_new duplicate node ID"
However, I have been using the same "startup.scd" for three years, and it occurs only in SC 3.9.

Steps to reproduce:
1. Save the following code block as "startup.scd"
```supercollider
s.options.numInputBusChannels = 10;
s.options.numOutputBusChannels = 10;
s.options.memSize = 2.pow(20);
s.waitForBoot(
	{
		s.plotTree;
		Window.allWindows.do{arg w; (w.name == "localhost Node Tree").if{w.bounds = Rect(0, 220-30, 300, Window.screenBounds.height-220)}};
		FreqScope(400, 192);
		s.scope(2).view.bounds= Rect(483, 4, 288, 220);
		ServerMeter(s).window.bounds= Rect(772, 0, 432, 220);
		Server.local.makeGui.window.bounds= Rect(1205, 0, 288, 98);
	}
)
```
2. run any synth.

Actual result:
Synth works without failure after four times.
Post window displays the following message:
"FAILURE IN SERVER /s_new duplicate node ID" 

Expected result:
Synth must work without server failure.

**Comment by brianlheim** [comment]
I think this is a dupe of #3272, and was fixed in #3273. @prko, can you confirm this?

**Comment by brianlheim** [comment]
also fyi you can use triple backticks for code blocks. i've edited the submission post so that it's a little more readable :)

**Comment by prko** [comment]
Yes, this is a dupe of #3262 and fixed already in #33273.
Thank you very much for your kind response and the information on the "triple backticks".

**Comment by brianlheim** [comment]
Thank you for the report! Always better to have dupes than silence :)
